.github/workflows/ci.yml

```diff
@@ -8,15 +8,22 @@ jobs:
     steps:
     - uses: actions/checkout@v3
 
-    - name: Clone Flutter SDK
-      # We can't do a depth-1 clone, because we need the most recent tag
-      # so that Flutter knows its version and sees the constraint in our
-      # pubspec is satisfied.  It's uncommon for flutter/flutter to go
-      # more than 100 commits between tags.  Fetch 1000 for good measure.
+    - name: Set up Homebrew
+      id: set-up-homebrew
+      uses: Homebrew/actions/setup-homebrew@master
+
+    - name: Install FVM
+      run: |
+        brew tap leoafarias/fvm
+        brew install fvm
```

Comment:
This is fairly slow — this step took 27s in the latest job recorded at https://github.com/zulip/zulip-flutter/pull/577/checks . That seems to be because it's building `fvm` from source:

```
==> Downloading https://github.com/leoafarias/fvm/archive/3.1.1.tar.gz
==> Downloading from https://codeload.github.com/leoafarias/fvm/tar.gz/refs/tags/3.1.1
==> Installing fvm from leoafarias/fvm
==> /home/linuxbrew/.linuxbrew/Cellar/fvm/3.1.1/libexec/bin/dart pub get
==> /home/linuxbrew/.linuxbrew/Cellar/fvm/3.1.1/libexec/bin/dart compile exe -Dversion=3.1.1 bin/main.dart -o fvm
🍺  /home/linuxbrew/.linuxbrew/Cellar/fvm/3.1.1: 1,022 files, 572.2MB, built in 20 seconds
```

If we instead do the `curl|bash` command [suggested as an alternative](https://fvm.app/documentation/getting-started/installation) in the docs:
```
$ curl -fsSL https://fvm.app/install.sh | bash
```
then on my machine that takes about 1.5s. The basic reason is that it's downloading a published release artifact rather than a source tree:
```bash
# Download FVM
URL="https://github.com/leoafarias/fvm/releases/download/$FVM_VERSION/fvm-$FVM_VERSION-$OS-$ARCH.tar.gz"
if ! curl -L "$URL" -o fvm.tar.gz; then
```

I don't love running `curl|bash` installs. For this one, I downloaded the script first, read it, then ran it. But ultimately it's not any different from what `brew tap leofarias/fvm; brew install fvm` is doing — both are taking a script published by the same person, and running it.